packages/example-completeness/src/utils.js

```diff
@@ -18,9 +17,11 @@ exports.getRequiredLocalizations = function(product) {
   return _.uniq(localizations);
 };
 
-exports.normalize = function(product, localization) {
+exports.normalize = (product, localization) => {
+  // eslint-disable-next-line
   delete product.masterData.current;
```

Comment:
This is mutating things which can/should be replaced with an object-spread so `{ ...previousObject, ...nextObject }`.